features/forgot_password.feature

```diff
@@ -0,0 +1,16 @@
+Feature:
+  A working "Forgot-Password" mechanism
+
+  Scenario: A User starts the "Forgot-Password" Routine
+    Given a new user ulf with email ulf@ulf.de
```

Comment:
You might want to create a step "a user [name] with email [email]" instead, so far the terminology has been user for an already stored user and new user for a user not in the database.

Reply:
:+1: